src/plugins/plugin.legend.js

```diff
@@ -483,33 +511,26 @@ var Legend = Element.extend({
 		}
 
 		// Chart event already has relative position in it
-		var x = e.x;
-		var y = e.y;
+		hoveredItem = me._getLegendItemAt(e.x, e.y);
```

Comment:
nit, I would rename this to just `item` or `legendItem`,  because it can be clicked item too.

Reply:
I see what you mean, although the item is always "hovered" in the sense that the mouse is currently hovered over it, even if the event we're handling is a `click` and not a `mousemove`.